### DISCUSSION TITLE
Wondering about estimated costs from this setup

### DISCUSSION THREAD

**Discussion by paustint** [commitdev/zero#412] [Q&A] [answered]
I am currently using a PaaS (render.com) which is working well for me, but I am intrigued by this setup and am interested in giving it a shot.

What would be a typical baseline monthly cost for a setup like this for something fairly simple with <TB bandwidth?

I am trying to understand (ballpark) if it is comparable to what I am paying now.

**Comment by bmonkman** [comment, marked as answer]
Hey there, glad to hear you're intrigued!
We have an estimated cost breakdown in the [infrastructure module readme](https://github.com/commitdev/zero-aws-eks-stack/#how-much-does-this-stack-cost) that works out to about $150/month for an environment that includes all the core infrastructure. For many startups, we've found it to be pretty reasonable when factoring in the availability of AWS credits.
If you want to chat more or would like a hand getting started, feel free to drop by our [community Slack](https://slack.getzero.dev)!

**Reply by paustint** [reply]
@bmonkman - Thanks! that is exactly what I was looking for. I might spin it up just to try it out, but it is at least 3x more that what I have my my current PaaS provider, so probably not ready to make a true move the the big person architecture just yet. :D

**Reply by bmonkman** [reply]
😂 No worries, feel free to reach out if you get to that point! I'll also drop a note here if we make any big cost-savings changes.
